src/models/factcheck.py

```
@@ -1,588 +1,28 @@
+    verdict: str
+    confidence: float = Field(ge=0.0, le=1.0)
+    explanation: str
+    sources: List[Source]
+
+class FactCheckOptions(BaseModel):
+    min_check_worthiness: float = Field(default=0.7, ge=0.0, le=1.0)
+    domains: List[str] = Field(default=["politics", "health"])
+    max_claims: int = Field(default=5, gt=0)
+    explanation_detail: str = Field(default="detailed")
+
+class FactCheckRequest(BaseModel):
```

Ruff (0.11.9)

Missing docstring in public class

(D101)